pattern-1/undeploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+oc delete -f activemq-service.yaml
+oc delete -f activemq-deployment.yaml
+oc delete -f key-manager-service.yaml
+oc delete -f key-manager-deployment.yaml
+oc delete -f api-core-service.yaml
+oc delete -f api-core-deployment.yaml
+oc delete -f api-gateway-deployment.yaml
```

Comment:
Shall we rename deploy and undeploy scripts as deploy-openshift,etc ? Also, shall we add a label called pattern=wso2apim-pattern-1 in every resource? Then we can remove resources in one command as in https://github.com/wso2/kubernetes-apim/blob/2.1.0/pattern-1/artifacts/undeploy-openshift.sh